lib/generator.js

```diff
@@ -390,8 +392,13 @@ class Generator {
    * Configure the templates based the desired renderer.
    */
   async configureTemplate() {
-    if (isReactTemplate(this.templateConfig)) {
-      await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
+    if(isReactTemplate(this.templateConfig)) {
+      if(this.compile) {
+        await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
+      } else {
+        // use the cached transpiled version provided by the template
+        this.templateContentDir = path.resolve(this.templateDir, TRANSPILED_TEMPLATE_LOCATION);
+      }
```

Comment:
Is the `else` section correct? I thought we are only going to do the transpilation when `this.compile` is set to be True. If I understand it correctly, the suggested change should be:

```suggestion
if (isReactTemplate(this.templateConfig) && this.compile) {
      await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
```

without a `else` clause, because we are not configuring it again if `this.compile` is set to be false.

Reply:
by default `compile` is set to false